src/vaults/VaultStorage.sol

```diff
@@ -72,9 +72,12 @@ abstract contract VaultStorageUnpadded {
     uint256 public token0FloorNum;
     /// @notice minimum return for TOKEN1 (out of `vault.DENOM`)
     uint256 public token1FloorNum;
+
+    /// @notice flag for enabling/disabling deposits
+    bool public depositsEnabled;
 }
 
 abstract contract VaultStorage is VaultStorageUnpadded {
     // @dev Padding 100 words of storage for upgradeability. Follows OZ's guidance.
-    uint256[100] private __gap;
+    uint256[99] private __gap;
```

Comment:
the 100 words of storage comment looks weird now. but we can't just change both to 99. maybe we add some comments explaining which storage was added in this upgrade